.github/workflows/ci.yml

```
@@ -0,0 +1,611 @@
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        cache: 'npm'
+        
+    - name: Install dependencies
+      run: npm ci --prefer-offline --no-audit
+      
+    - name: Decrypt and load environment variables
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:27: Double quote to prevent globbing and word splitting

(shellcheck)